### PR TITLE
big update

### PR DESCRIPTION
**Pull request by Marik33409**
all tables were updated, classes were renamed, all methods of recording through annotations are updated, cascade deletion of records from all tables works and tested

### CODE REVIEW

File: src/main/java/entity/PlanTasks.java [hunk 1]
```diff
@@ -6,11 +6,7 @@
 
 @Entity
 @Table(name = "plan_tasks", schema = "public", catalog = "ipr")
-public class PlanTasksEntity {
```
Comment (JerryngtonD) [inline src/main/java/entity/PlanTasks.java:9]: конструктор по умолчанию зачем убрал ?

**Review by JerryngtonD — COMMENTED** [review]
Связь OneToOne не нашел реализации

File: src/main/java/entity/PlanTasks.java (hunk 1 above)
Comment (Marik33409) [reply src/main/java/entity/PlanTasks.java:9]: Добавил конструкторы по умолчанию во все классы

**Comment by Marik33409** [comment]
OneToOne добавлена

**Pull request merged**